main.go

```diff
@@ -1,4 +1,4 @@
-package main // import "go.uber.org/sally"
```

Comment:
Ah, thanks for sharing the link. I couldn't find what that was. The reason I removed it was because I couldn't compile myself with this [error](https://github.com/golang/go/issues/13531).

I'll remove this from the commit.